lib/discord_bot/commands/slash/nice_message.rb

```diff
@@ -0,0 +1,38 @@
+module DiscordBot
+  module Commands
+    module Slash
+      class NiceMessage < Base
+        class << self
+          def description
+            "allows user to send a nice message to another user though ruby bot #{Config.bot_name}"
```

Comment:
`ruby bot` and `#{Config.bot_name}` are redundant.

Reply:
fixed